src/schema/v2/partner.ts

```diff
@@ -248,13 +259,20 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
             size: number
             total_count: boolean
             sort: string
+            status: string
+            day_threshold: number
           }
 
           const gravityArgs: GravityArgs = {
             total_count: true,
             page,
             size,
             sort: args.sort,
+            // default Enum value for status is not properly resolved
+            // so we have to manually resolve it by lowercasing the value
+            // https://github.com/apollographql/graphql-tools/issues/715
+            ...(args.status && { status: args.status.toLowerCase() }),
```

Comment:
Why the guard here, instead of just `status: args.status`, `day_threshold: args.dayThreshold` (akin to the other arguments)?

Reply:
Copying the logic from here: https://github.com/artsy/metaphysics/blob/master/src/schema/v2/city/index.ts#L89

Reply:
Removed the guard, not sure why it was initially used.

Reply:
It's possible there was some param where this was important to do, and then got copied elsewhere. Always be critical when 🍝 !